packages/next/src/build/flying-shuttle/detect-changed-entries.ts

```diff
@@ -64,6 +68,45 @@ export async function detectChangedEntries({
       },
     }
   }
+  const foundShuttleManifest: ReturnType<typeof getShuttleManifest> =
+    JSON.parse(
+      await fs.promises.readFile(
+        path.join(shuttleDir, 'shuttle-manifest.json'),
+        'utf8'
+      )
```

Comment:
i'd probably make this a `getShuttleManifest` util (which could also be used by `hasShuttle`) and then rename the current `getShuttleManifest` to `generateShuttleManifestFromConfig` or something like that